tests/test_helpers/test_tenants.py

```diff
@@ -287,3 +287,27 @@ def test_fix_lms_base_port(lms_root_port, domain_name, expected_result):
 def test_fix_lms_base_empty_domain_name(domain_name):
     """Verify that fix_lms_base sets the correct port for the result."""
     assert tenants.fix_lms_base(domain_name) == ''
+
+
+@pytest.mark.django_db
+def test_get_fx_dashboard_url():
+    """Verify _get_fx_dashboard_url reurns correct address"""
+    factory = RequestFactory()
+    request = factory.get('/some-path')
+    request.site = type('Site', (), {})()
+    request.site.name = 'test_site'
+    request.LANGUAGE_CODE = 'en'
+    tenant_info = {
+        'sites': {
+            1: 'test_site',
+            2: 'another_site',
+        }
+    }
+    # Test when tenant ID is accessible
+    with patch('futurex_openedx_extensions.helpers.tenants.get_all_tenants_info', return_value=tenant_info):
```

Comment:
Please rely on the `futurex-openedx-extensions` data mocks that @shadinaif have developed, this would simulate the request in a better way.

This test is mostly validating it's own implementation (`test_get_fx_dashboard_url`) rather than the actual (`get_fx_dashboard_url`) funciton behaviour.